cmd/aws-vpc-cni/main.go

```diff
@@ -445,7 +445,8 @@ func _main() int {
 		return 1
 	}
 
-	err = cp.CopyFile(tmpAWSconflistFile, defaultHostCNIConfDirPath+awsConflistFile)
+	hostCniConfDirPath := utils.GetEnv(envHostCniConfDirPath, defaultHostCNIConfDirPath)
```

Comment:
Nit: either replace `HostCni` -> `HostCNI` in envHost... or `HostCNI` -> `HostCni` in defaultHost..